src/app/pages/MainArticlesScreen/store/actions/index.ts

```diff
@@ -16,25 +16,33 @@ export const {
   failed: fetchArticlesFailed,
 } = reduxAsyncAction(MainScreenTypes.FETCH_ARTICLES);
 
+export type fetchArticlesType = ReturnType<typeof fetchArticles>;
+export type fetchArticlesSuccessType = ReturnType<typeof fetchArticlesSuccess>;
+export type fetchArticlesFailedType = ReturnType<typeof fetchArticlesFailed>;
+
 export const reloadArticles = (payload: Array<ArticleData>) =>
   ({
-    type: MainScreenTypes.RELOAD_ARTICLES,
+    type: MainScreenTypes.RELOAD_ARTICLES as string,
```

Comment:
is this really needed?

Reply:
On second thought I think it's not needed. 
I used it because of inconsistency in action types – actions declared via action generic are `string` type and the explicitly declared are `enum` type but it seems not to affect anything. So I'll delete it.